src/components/advanced-search/advanced-search-dates.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <v-dialog id="advanced-search-dates" v-model="dialog" persistent max-width="44rem">
+    <v-card height="32rem" class="py-6">
+      <v-row>
+        <v-col>
+          <span class="title-bold-16" :class="{ 'error-message': validate && !hasStartDate }">Select Start Date:</span>
+          <v-date-picker
+            id="start-date-calendar"
+            class="mt-2"
+            v-model="startDateText"
+            :max="currentDateString"
+            elevation="5"
+          >
+          </v-date-picker>
+        </v-col>
+        <v-col>
+          <span class="title-bold-16" :class="{ 'error-message': validate && !hasEndDate }">Select Start Date:</span>
+          <v-date-picker
```

Comment:
Oddly, in this date picker, when I click on today's day, the styling is different than any other day. I was confused at first whether today is a valid value. Can this be changed?

![image](https://user-images.githubusercontent.com/10002889/117895766-e9e4c800-b273-11eb-914c-7c04e9fec1a0.png)

Reply:
hmm let me look into this, i didn't notice before!

Reply:
Fixed! It is like this in the shared component, however here it was still a light gray opposed to the light blue. 
Updated tho, good eye 👍 

![Screen Shot 2021-05-12 at 8 10 33 AM](https://user-images.githubusercontent.com/53542131/117999276-87cea600-b2f9-11eb-8d7b-aae9e66ca64c.png)

Reply:
Much better thanks!

Reply:
I just go through it and see if it looks/functions as I, an experienced web user, expect. It's always easier reviewing someone else's code 😁

Reply:
Totally! Makes for a very good review as well, albeit a little more work.